javaslang/generator/Generator.scala

```diff
@@ -3522,16 +3522,16 @@ object Generator {
 
     // returns i as ordinal, i.e. 1st, 2nd, 3rd, 4th, ...
     def ordinal: String =
-      if (i / 10 == 1) {
-        s"${i}th"
+      s"$i" + (if (i >= 4 && i <= 20) {
+        "th"
       } else {
         i % 10 match {
-          case 1 => "1st"
```

Comment:
Right, this looks better. could you please copy this updated code also to javaslang-test/generator/Generator.scala? Currently we have redundant code :-//

I will change the Generator in a (near?) future. Currently the maven build process hinders us from having one general Generator import.